components/compositing/windowing.rs

```diff
@@ -200,6 +200,13 @@ pub struct EmbedderCoordinates {
 }
 
 impl EmbedderCoordinates {
+    // Get the viewport for webrender usage.
```

Comment:
Rustdoc comments should start with three `/` so this should be something like:

```suggestion
    /// Get the unflipped viewport rectangle for use with the WebRender API.
```